build.sh

```diff
@@ -86,6 +86,7 @@ BuildWithXbuild()
     export MONO_IOMAP=case
     CheckExitCode msbuild /p:Configuration=Debug /t:Clean $slnFile
     CheckExitCode msbuild /p:Configuration=Release /t:Clean $slnFile
+    mono $nuget locals all -clear
```

Comment:
Do we need to also do this on the windows side. I know it works now, just wondering if it would help. Also will this hurt anything when run local?

Reply:
I've added this to the windows case, along with a clean for the Debug configuration, like we have in mono.